error repeated many times in console, hard to read output

Write aqua, make a mistake. Expected that each error is only once. Actual - error repated many times (seems as many times as function reused in other aquas)
```
2021.10.22 19:48:16 [INFO] Aqua Compiler 0.3.2-233
2021.10.22 19:48:20 [ERROR] ---- Error: Test.aqua:46:40
46         peers_all <- Op.concat(my, me, his, nil)
                                          ^^^=======
                                          Types mismatch, expected: ?[]string, given: []string

2021.10.22 19:48:20 [ERROR] ---- Error: Test.aqua:71:13
71             records: *u8
               ^^^^^^^======
               This name was already defined in the scope

2021.10.22 19:48:20 [ERROR] ---- Error: Test.aqua:46:40
46         peers_all <- Op.concat(my, me, his, nil)
                                          ^^^=======
                                          Types mismatch, expected: ?[]string, given: []string

2021.10.22 19:48:20 [ERROR] ---- Error: Test.aqua:71:13
71             records: *u8
               ^^^^^^^======
               This name was already defined in the scope

2021.10.22 19:48:20 [ERROR] ---- Error: Test.aqua:46:40
46         peers_all <- Op.concat(my, me, his, nil)
                                          ^^^=======
                                          Types mismatch, expected: ?[]string, given: []string

2021.10.22 19:48:20 [ERROR] ---- Error: Test.aqua:71:13
71             records: *u8
               ^^^^^^^======
               This name was already defined in the scope

2021.10.22 19:48:20 [ERROR] ---- Error: Test.aqua:46:40
46         peers_all <- Op.concat(my, me, his, nil)
                                          ^^^=======
                                          Types mismatch, expected: ?[]string, given: []string

2021.10.22 19:48:20 [ERROR] ---- Error: Test.aqua:71:13
71             records: *u8
               ^^^^^^^======
               This name was already defined in the scope

2021.10.22 19:48:20 [ERROR] ---- Error: Test.aqua:46:40
46         peers_all <- Op.concat(my, me, his, nil)
                                          ^^^=======
                                          Types mismatch, expected: ?[]string, given: []string

2021.10.22 19:48:20 [ERROR] ---- Error: Test.aqua:71:13
71             records: *u8
               ^^^^^^^======
               This name was already defined in the scope

2021.10.22 19:48:20 [ERROR] ---- Error: Test.aqua:46:40
46         peers_all <- Op.concat(my, me, his, nil)
                                          ^^^=======
                                          Types mismatch, expected: ?[]string, given: []string

2021.10.22 19:48:20 [ERROR] ---- Error: Test.aqua:71:13
71             records: *u8
               ^^^^^^^======
               This name was already defined in the scope

2021.10.22 19:48:20 [ERROR] ---- Error: Test.aqua:46:40
46         peers_all <- Op.concat(my, me, his, nil)
                                          ^^^=======
                                          Types mismatch, expected: ?[]string, given: []string

2021.10.22 19:48:20 [ERROR] ---- Error: Test.aqua:71:13
71             records: *u8
               ^^^^^^^======
               This name was already defined in the scope


```

--- a/aqua-src/import-empty.aqua
+++ b/aqua-src/import-empty.aqua
@@ -1,6 +1,7 @@
 -- import.aqua
 module Import.Test
 import foo from "export"
+import someFunc from "ret"
 
 use foo as f from "export" as Exp
 

--- a/cli/src/main/scala/aqua/AquaPathCompiler.scala
+++ b/cli/src/main/scala/aqua/AquaPathCompiler.scala
@@ -5,21 +5,18 @@ import aqua.compiler.{AquaCompiled, AquaCompiler, AquaError}
 import aqua.files.{AquaFileSources, FileModuleId}
 import aqua.io.*
 import aqua.model.transform.TransformConfig
-import aqua.parser.{Ast, LexerError}
-import aqua.parser.lift.FileSpan
-import cats.data.*
-import cats.syntax.functor.*
-import cats.syntax.applicative.*
-import cats.syntax.show.*
-import cats.{~>, Applicative, Eval, Monad, Show}
-import scribe.Logging
-import fs2.io.file.{Files, Path}
-import aqua.parser.lift.{LiftParser, Span}
-import cats.parse.LocationMap
-import cats.~>
 import aqua.parser.lift.LiftParser.LiftErrorOps
-import Span.spanLiftParser
-import aqua.parser.Parser
+import aqua.parser.lift.Span.spanLiftParser
+import aqua.parser.lift.{FileSpan, LiftParser, Span}
+import aqua.parser.{Ast, LexerError, Parser}
+import cats.data.*
+import cats.parse.LocationMap
+import cats.syntax.applicative.*
+import cats.syntax.functor.*
+import cats.syntax.show.*
+import cats.{Applicative, Eval, Monad, Show, ~>}
+import fs2.io.file.{Files, Path}
+import scribe.Logging
 
 object AquaPathCompiler extends Logging {
 
@@ -48,7 +45,7 @@ object AquaPathCompiler extends Logging {
         transformConfig,
         targetPath.map(sources.write).getOrElse(dry[F])
       )
-      .map(_.leftMap(_.map(_.show)))
+      .map(_.leftMap(_.map(_.show).distinct))
   }
 
   def dry[F[_]: Applicative](

--- a/cli/src/main/scala/aqua/AquaPathCompiler.scala
+++ b/cli/src/main/scala/aqua/AquaPathCompiler.scala
@@ -45,6 +45,7 @@ object AquaPathCompiler extends Logging {
         transformConfig,
         targetPath.map(sources.write).getOrElse(dry[F])
       )
+      // 'distinct' to delete all duplicated errors
       .map(_.leftMap(_.map(_.show).distinct))
   }
 

